src/m_menu.c

```diff
@@ -1331,7 +1331,8 @@ void M_QuitResponse(int ch)
 {
   if (ch != 'y')
     return;
-  if ((!netgame || demoplayback) // killough 12/98
+  if (show_endoom && // play quit sound only if showing ENDOOM
```

Comment:
Actually, `show_endoom` may even mean to only show ENDOOM if provided by a PWAD.

Reply:
Oops, let me start over.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/923/commits/999f9bae4f0a70f55d8dd18366443241060663c0